src/weapon.py

```diff
@@ -161,7 +164,7 @@ def update(
 
     def shoot(self, projectile_list: arcade.SpriteList):
         bullet = arcade.SpriteSolidColor(10, 5, arcade.color.RED)
-        bullet.position = self.weapon_sprite.position
+        bullet.position = add_vec2(self.weapon_sprite.position, self.muzzle_offset)
```

Comment:
This does not account for rotation of the car (see other comment)

Reply:
Fixed